tests/app/v2/notifications/test_notification_schemas.py

```diff
@@ -133,21 +136,24 @@ def test_get_notifications_request_invalid_statuses_and_template_types():
 }
 
 
-@pytest.mark.parametrize("input", [valid_phone_number_json, valid_recipient_identifier_json,
-                                   valid_phone_number_and_recipient_identifier_json, valid_json_with_optionals])
-def test_post_sms_schema_is_valid(input):
-    assert validate(input, post_sms_request_schema) == input
+@pytest.mark.parametrize("data", [
+    valid_phone_number_json,
+    valid_recipient_identifier_json,
+    valid_phone_number_and_recipient_identifier_json,
+    valid_json_with_optionals,
+])
+def test_post_sms_schema_is_valid(data):
+    assert validate(data, post_sms_request_schema) == data
 
 
-@pytest.mark.parametrize("template_id",
-                         ['2ebe4da8-17be-49fe-b02f-dff2760261a0' + "\n",
-                          '2ebe4da8-17be-49fe-b02f-dff2760261a0' + " ",
-                          '2ebe4da8-17be-49fe-b02f-dff2760261a0' + "\r",
-                          "\t" + '2ebe4da8-17be-49fe-b02f-dff2760261a0',
-                          '2ebe4da8-17be-49fe-b02f-dff2760261a0'[4:],
-                          "bad_uuid"
-                          ]
-                         )
+@pytest.mark.parametrize("template_id", [
+    "2ebe4da8-17be-49fe-b02f-dff2760261a0\n",
+    "2ebe4da8-17be-49fe-b02f-dff2760261a0 ",
+    "2ebe4da8-17be-49fe-b02f-dff2760261a0\r",
+    "\t2ebe4da8-17be-49fe-b02f-dff2760261a0",
+    "4da8-17be-49fe-b02f-dff2760261a0",
+    "bad_uuid",
+])
```

Comment:
trying not to say the same thing for all the tests (so I skipped a few examples above), but this is a perfect example of using IDs for parameterize:` ids=["new line", "trailing space", "return", "tab indent", "too few chars", "string not uuid"] `